packages/airnode-node/src/index.ts

```diff
@@ -3,5 +3,8 @@ export * as config from './config';
 export * as providerState from './providers/state';
 export * as logger from './logger';
 export * as promiseUtils from './utils/promise-utils';
+export * as evm from './evm';
+export * as adapterHttpParameters from './adapters/http/parameters';
```

Comment:
This file is a bit messy and I think we've just been dumping stuff here without much consideration. We're exporting things like `promiseUtils` and `utils` which will export duplicate functions.

I think this it rather should be something like:

```ts
// src/index.ts
export * as adapters from './adapters';

// src/adapters/index.ts
export * as http from './http';

// src/adapters/http/index.ts
export * as parameters from './parameters';
```

This way, you would access the parameters functions by doing:

```ts
import { adapters } from '@airnode-node';

adapters.http.parameters.somefunction();
```

I think this is a lot cleaner (especially with naming) and allows us to easily grow what we export. What do you think?

Reply:
Yes, that is a much cleaner approach. Thanks!